app/frontend/src/AppRoutes.tsx

```diff
@@ -17,8 +21,10 @@ export const loginPasswordRoute = `${loginRoute}/password`;
 
 export const signupRoute = "/signup";
 export const profileRoute = "/profile";
+export const editProfileRoute = "/profile/edit";
+export const editHostingPreferenceRoute = "/hosting-preference/edit";
```

Comment:
Will probably have to change this later but it's good for now

Reply:
What do you mean by that? As in you might put them both on the same page again?

Reply:
No we just need to sort out the route hierarchy and naming

Reply:
Ahh yeah, `profile` was clear but wasn't sure about `hosting-preference` name itself. I think the convention is ok since `kebab-case` is probably the most commonly used one for URLs